integration/examples/getting-started/Dockerfile

```diff
@@ -1,6 +1,11 @@
 FROM golang:1.12.9-alpine3.10 as builder
+
+# These flag values are provided when building for debug
+ARG GO_GCFLAGS
```

Comment:
`GO_GCFLAGS` and `GO_LDFLAGS` are explicitly added as dockerfile args and their values are provided by Skaffold